icu4j/pom.xml

```diff
@@ -114,7 +114,7 @@
   <licenses>
     <license>
       <name>Unicode-3.0</name>
-      <url>https://raw.githubusercontent.com/unicode-org/icu/main/LICENSE</url>
+      <url>https://raw.githubusercontent.com/unicode-org/icu/refs/heads/maint/maint-${icu.major.version}/LICENSE</url>
```

Comment:
FYI -- it seems to work just as well without the `/refs/heads` part, but when clicking the "raw" button on a regular file view, GitHub does include that part.